README.md

```diff
@@ -163,7 +163,7 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
 
 ## Usage
 
-> ❗️ If you're migrating from older SwiftGen versions, don't forget to [read the Migration Guide](https://github.com/SwiftGen/SwiftGen/blob/master/Documentation/MigrationGuide.md).
+> ❗️ If you're migrating from older SwiftGen versions, don't forget to [read the Migration Guide](./Documentation/MigrationGuide.md).
```

Comment:
There's no need for the `./` at the start of that URL, they work fine without it. Also applies to those relative URL changes below.

Reply:
Yeah, I just saw that we already had multiple relative URLs starting with `./` elsewhere in existing documentation, so I wanted to uniformize, but I agree we should uniformize in the other direction, dropping the `./` in all relative links (even existing ones outside of this PR)